internal/k8s/reconciler/validator/route_test.go

```diff
@@ -52,7 +51,8 @@ func TestRouteValidate(t *testing.T) {
 		},
 	})
 	require.NoError(t, err)
-	require.True(t, routeState.ResolutionErrors.Empty())
+	require.NotNil(t, routeState)
+	assert.True(t, routeState.ResolutionErrors.Empty())
```

Comment:
Changes in this file are just improving our test hygiene based on a linter run that I did locally